vpr/src/analytical_place/partial_placement.cpp

```diff
@@ -66,10 +67,13 @@ double PartialPlacement::estimate_post_placement_wirelength(const APNetlist& net
 
         // Floor the positions to get the x and y coordinates of the tiles each
         // block belongs to.
-        unsigned tile_dx = std::floor(max_x) - std::floor(min_x);
-        unsigned tile_dy = std::floor(max_y) - std::floor(min_y);
+        // TODO: In the placer, the x and y dimensions are multiplied by cost
+        //       factors based on the channel width. Should somehow bring these
+        //       in here.
```

Comment:
Probably don't want to bring those into a hpwl estimate but could bring them into a cost estimate.